modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// *networking/allocating-load-balancers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-allocating-load-balancers-to-subnets-procedure_{context}"]
+= Allocate API and Ingress Load Balancers to Specific Subnets
+
+Perform the following steps to allocate API and Ingress load balancers to specific subnets.
+
+.Prerequisites
+
+. Log in as a cluster administrator.
+. Ensure that your cluster is deployed on a supported cloud platform, such as AWS. 
+//Note to self: Is above prerequisite correct? Or can you only do this procedure on cluster deployed on AWS? The Jira didn't make that clear.
+. Obtain the subnet ID.
+
+.Procedure
+
+. Prepare the IngressController YAML file. 
+
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+Metadata:
+name: example-ingress
+namespace: openshift-ingress-operator
+Spec:
+endpointPublishingStrategy:
+ 		type: LoadBalancerService
+ 		loadBalancer:
+scope: External
+subnet: "<your-subnet-id>"  # Specify the target subnet for the load balancer <1>
+----
+<1> Add the `subnet` field under the load balancer configuration in your YAML file. Replace `<your-subnet-id>` with the actual subnet identifier.
+
+. Use the OpenShift CLI to apply the YAML file:
+
+[source,terminal]
+----
+oc apply -f your-ingresscontroller.yaml
+----
+
+. Verify Subnet allocation.  
+.. List the services in the ingress namespace.
+
+[source,terminal]
+----
+oc get svc -n openshift-ingress
+----
+
+.. Confirm that the annotations show the service that you specified.
+
+[source,terminal]
+----
+oc describe svc <service-name> -n openshift-ingress
+----
+
+. Check your cloud provider’s console or CLI tools to ensure the frontend IP is allocated from the specified subnet.
```

Comment:
There can be multiple subnets and multiple IPs for a LB DNS name. It may be not very straight forward to find those IPs and to check that they are from the right subnet CIDRs. If we suggest the user to check on AWS, they should check that all the specified subnets are linked to the load balancer. On AWS console, for instance, it's visible on the main page of a given load balancer.